### PR TITLE
Scope under MetaMask org

### LINKED ISSUE #52: Scope package under the `@metamask/` npm org
Deprecate `eth-rpc-errors` and rename this package to `@metamask/errors`.

### PR DESCRIPTION
**Pull request by FrederikBolding**
Scopes the package under the MetaMask npm org.

Fixes #52

### CODE REVIEW

**Comment by Mrtenz** [comment]
How is this breaking?

File: package.json [hunk 1]
```diff
@@ -1,9 +1,8 @@
 {
-  "name": "eth-rpc-errors",
+  "name": "@metamask/eth-rpc-errors",
   "version": "4.0.3",
   "description": "Ethereum RPC and Provider errors.",
   "license": "MIT",
-  "author": "Erik Marks <rekmarks@protonmail.com>",
```
Comment (Mrtenz) [inline package.json:6]: Why?

**Comment by FrederikBolding** [comment]
> How is this breaking?

We should deprecate the old namespace and point it at the new one.

File: package.json (hunk 1 above)
Comment (FrederikBolding) [reply package.json:6]: We don't usually have authors on MetaMask packages and we will be rewriting part of it regardless

**Review by Mrtenz — DISMISSED** [review]

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**